rosbag2_storage_default_plugins/include/rosbag2_storage_default_plugins/sqlite/sqlite_exception.hpp

```diff
@@ -18,10 +18,17 @@
 #include <stdexcept>
 #include <string>
 
+#include "rosbag2_storage_default_plugins/visibility_control.hpp"
+
+#ifdef _WIN32
+# pragma warning(push)
+# pragma warning(disable:4275)
```

Comment:
what's that warning? Can you maybe put a little comment on this to trace it later on?